apps/api/src/app/app.service.ts

```diff
@@ -5,6 +5,11 @@ import { IArchie } from '@archmaster/shared/types';
 export class AppService {
   getArchie(): IArchie {
```

Comment:
Maybe we should use a more generic name like `getCuteDog` 😛  `getDog`

Reply:
we can change it when implementing the controllers and services. Which in turn will depend on how the contract between FE and BE should look like.